ref/general/authorization.adoc

```diff
@@ -13,17 +13,22 @@
 :page-type: general
 = Authorization in Open Liberty
 
-In order for an application to be secure, it must be able to manage access to protected resources. Authorization determines which resources a user can access.
+For an application to be secure, it must be able to manage access to protected resources. Authorization determines which resources a user can access.
 
-In an application where only certain users or groups are entitled to access certain resources, users and groups are assigned roles that represent which resources they can access. For example, in a payroll application, users in the `Employee` role can view a page that lists their hourly wages and hours worked. However, only users in the `Manager` role can make changes to employee wages. During link:/docs/ref/general/#authentication.html[authentication], Open Liberty identifies users, what groups they might be in, and which roles they have. When a user attempts to access a protected resource on the server, this information is used to determine whether the user is authorized to access the resource.
+In an application where only certain users or groups are entitled to access certain resources, users and groups are assigned roles that represent which resources they can access. For example, in a payroll application, users in the `Employee` role can view a page that lists their hourly wages and hours worked. However, only users in the `Manager` role can edit employee wages. During link:/docs/ref/general/#authentication.html[authentication], Open Liberty identifies users, what groups they might be in, and which roles they have. When a user attempts to access a protected resource on the server, this information is used to determine whether the user is authorized to access the resource.
 
+In the following diagram, a user requests access to protected resources in an application. During authentication (steps 1-4), the user's identity and security roles are confirmed. During authorization (steps 5-6), the roles that specified for the user are checked against the roles that are defined for the application resources. The user is granted access to any resources that match their role.
```

Comment:
Update
"...the roles that **are** specified for the user..."

Reply:
fixed- thanks